tutor/commands/upgrade/compose.py

```diff
@@ -168,7 +179,17 @@ def upgrade_mongodb(
         )
         return
 
+    # Just need the main release version for certain checks
+    mongo_version = int(to_docker_version[0])
+    admin_command: t.Dict[str, t.Any] = {
+        "setFeatureCompatibilityVersion": to_compatibility_version
+    }
+    if mongo_version == 7:
+        # Explicit confirmation is required to upgrade to 7 from 6
+        # https://www.mongodb.com/docs/manual/reference/command/setFeatureCompatibilityVersion/#confirm
+        admin_command.update({"confirm": 1})
```

Comment:
Just making sure I understand what this means: this also works in headless, non-interactive mode, right? I mean that the user is not explicitly asked to manually confirm by typing "y" on their keyboard, right?